Formula/o/oatpp.rb

```diff
@@ -0,0 +1,42 @@
+class Oatpp < Formula
+  desc "Light and powerful C++ web framework"
+  homepage "https://oatpp.io/"
+  url "https://github.com/oatpp/oatpp/archive/refs/tags/1.3.1.tar.gz"
+  sha256 "9dd31f005ab0b3e8895a478d750d7dbce99e42750a147a3c42a9daecbddedd64"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV.append "LDFLAGS", "-Wl,-rpath,#{lib}/oatpp-#{version}" if OS.linux?
+
+    # Remove in the next release.
+    # See: https://github.com/oatpp/oatpp/issues/988#issuecomment-2525575710
+    inreplace "src/oatpp/core/base/Environment.hpp", "1.3.0", version.to_s
```

Comment:
any issue of tracking the next release? (the master branch also has the cmake-4 patch now)

Reply:
Don't see one, but can resubmit then if that's preferred

Reply:
not really, I did not see much issues of shipping the PR but just more curious :)